inc/Engine/Media/Fonts/Context/Context.php

```diff
@@ -16,9 +16,7 @@ class Context extends AbstractContext {
 	public function is_allowed( array $data = [] ): bool {
 		$is_allowed = $this->run_common_checks(
 			[
-				'do_not_optimize' => false,
-				'bypass'          => false,
```

Comment:
Why do we remove those? I don't believe the optimization should be applied when using the bypass or donotoptimize

Reply:
Sorry @remyperona didn't notice ur comment, we need to remove those to work with SaaS visits because SaaS visits the page with those query strings:
```
/?nowprocket=1&no_optimize=1&wpr_imagedimensions=1
```

And we call this method `rewite_fonts` with `rocket_performance_hints_buffer` which is fired only when `wpr_imagedimensions` query string is there.

Do u see any issue when using that with rocket_buffer?

Reply:
Not having those checks is going to apply the optimization when using the `nowprocket` parameter or the `DONOTROCKETOPTIMIZE` constant

Reply:
Ok, I have an idea to fix that, let me test it

Reply:
Now I separated context for normal optimizations from SaaS visits optimizations.